index.html

```diff
@@ -45,6 +45,183 @@ <h1 class="headline">
                 </ul>
             </div>
         </section>
+        <main class="project-section">
```

Comment:
- [ ] [REQUIRED] Kindly, fix the space issue as requested by previous reviewer.

![image](https://user-images.githubusercontent.com/60979926/170658324-59fdcbcc-42dd-40fb-a20b-76db76c91826.png)